src/contractTest/java/uk/gov/hmcts/reform/sscs/consumer/HearingPostConsumerTest.java

```diff
@@ -0,0 +1,92 @@
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.cloud.openfeign.EnableFeignClients;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.test.context.ActiveProfiles;
+import uk.gov.hmcts.reform.sscs.ContractTestDataProvider;
+import uk.gov.hmcts.reform.sscs.model.single.hearing.HearingResponse;
+import uk.gov.hmcts.reform.sscs.service.HmcHearingApi;
+import uk.gov.hmcts.reform.sscs.utility.BasePactTest;
+
+@ExtendWith(PactConsumerTestExt.class)
+@EnableFeignClients(basePackages = {"uk.gov.hmcts.reform.sscs.service"})
+@ActiveProfiles("contract")
+@SpringBootTest
+@PactTestFor(port = "10000")
+@PactFolder("pacts")
+public class HearingPostConsumerTest extends ContractTestDataProvider {
+
+    @Autowired
+    private HmcHearingApi hmcHearingApi;
+
+    @Pact(consumer = CONSUMER_NAME)
+    public RequestResponsePact createHearingRequestForValidRequest(PactDslWithProvider builder) {
+        return builder.given(CONSUMER_NAME + " successfully creating hearing request ").uponReceiving(
+            "Request to create hearing request to save details")
+            .path(PATH_HEARING).method(HttpMethod.POST.toString()).body(
+                toJsonString(generateHearingRequest())).headers(headers).willRespondWith()
+            .status(HttpStatus.OK.value()).body(
+                BasePactTest
+                .generatePostHearingsJsonBody(MSG_200_POST_HEARING))
+            .toPact();
+    }
+
+    @Pact(consumer = CONSUMER_NAME)
+    public RequestResponsePact validationErrorFromPostHearing(PactDslWithProvider builder) {
+        return builder.given(CONSUMER_NAME
+                                 + " throws validation error while trying to create hearing")
+                .uponReceiving("Request to CREATE hearing for invalid hearing request")
+                    .path(PATH_HEARING).method(HttpMethod.POST.toString())
+            .body(toJsonString(generateInvalidHearingRequest())).headers(headers)
+            .willRespondWith().status(HttpStatus.BAD_REQUEST.value())
+            .body(new PactDslJsonBody().stringType(FIELD_MESSAGE, MSG_400_POST_HEARING)
+                      .stringValue(FIELD_STATUS, BAD_REQUEST)
+                          .eachLike(FIELD_ERRORS, 1)
+                      .closeArray())
+            .toPact();
+    }
+
+    @Test
+    @PactTestFor(pactMethod = "createHearingRequestForValidRequest")
+    public void shouldSuccessfullyPostHearingRequest() throws JsonProcessingException {
+        HearingResponse hearingResponse = hmcHearingApi.createHearingRequest(
+            IDAM_OAUTH2_TOKEN,
+            SERVICE_AUTHORIZATION_TOKEN,
+            generateHearingRequest()
+        );
+
+        Assertions.assertTrue(hearingResponse.getHearingRequestId() > ZERO_LENGTH);
+        Assertions.assertFalse(hearingResponse.getStatus().isEmpty());
```

Comment:
Can use import static for assertTrue and assertFalse since there is only 2.

Reply:
Actually it was asked in review comment to use static import, so used